src/title/titleSelectors.js

```diff
@@ -16,12 +16,15 @@ import { NULL_SUBSCRIPTION } from '../nullObjects';
  *
  * @param narrow - Narrow of the screen, if is is chat screen else undefined
  */
-export const getTitleBackgroundColor = (narrow: Narrow) =>
+export const getTitleBackgroundColor = (narrow?: Narrow) =>
   createSelector(
     getSubscriptions,
     subscriptions =>
+      // Prettier messes up the ternary so the linter complains.
+      // prettier-ignore
       isStreamOrTopicNarrow(narrow)
-        ? (subscriptions.find(sub => narrow[0].operand === sub.name) || NULL_SUBSCRIPTION).color
+         // $FlowFixMe: isStreamOrTopicNarrow will make sure that narraw[0] exists
```

Comment:
I tried, but can't find a simple solution. Can you please help me out? :)

Reply:
Here's a strategy for solving this kind of problem:
* OK, you're confident that the actual values here will always be a certain way (`narrow` will be an `Array`).
* Why are you confident of that?
* Make the code explicit about that reasoning.

In this case:
* You're confident things are OK because `narrow` is an `Array`. Why's that?
* If I understand you right, it's because `isStreamOrTopicNarrow` checks `Array.isArray(narrow)`.
* OK, we can check that here too:
```
subscriptions =>
  Array.isArray(narrow) && isStreamOrTopicNarrow(narrow)
    ? ...
```
   Or maybe just say `narrow &&`.

Reply:
Tried with 
```
subscriptions =>
  Array.isArray(narrow) && isStreamOrTopicNarrow(narrow)
    ? ...
```
and
```
narrow &&
```
it didn't fixed, it was giving:
```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ src/title/titleSelectors.js:26:45

Cannot get narrow[0] because an indexer property is missing in undefined [1].

 [1] 19│ export const getTitleBackgroundColor = (narrow?: Narrow) =>
     20│   createSelector(
     21│     getSubscriptions,
     22│     subscriptions =>
     23│       // Prettier messes up the ternary so the linter complains.
     24│       // prettier-ignore
     25│       Array.isArray(narrow) && narrow.length > 0 && isStreamOrTopicNarrow(narrow)
     26│         ? (subscriptions.find(sub => narrow[0].operand === sub.name) || NULL_SUBSCRIPTION).color
     27│         : 'transparent',
     28│   );
     29│

```

Fixed it via this:
```
 isStreamOrTopicNarrow(narrow)
        ? (
            subscriptions.find(sub => Array.isArray(narrow) && narrow[0].operand === sub.name)
            || NULL_SUBSCRIPTION
          ).color
        : 'transparent',
```

:)